lib/gmaps.py

```diff
@@ -188,7 +185,7 @@ def get_confidence(data, cfg):
             if all([loc in locations[hash]["locations"] for loc in locations[hash2]["locations"]]):
                 to_del.append(hash2)
     for hash in to_del:
-        del(locations[hash])
+        del (locations[hash])
```

Comment:
Nice, I always thought it was a function, I learn something

Reply:
I just removed the brackets as well :)